accounting-service/accounting-api/build.gradle

```diff
@@ -1,29 +1,17 @@
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.1.4.RELEASE")
-    }
+plugins {
+    id "com.jfrog.artifactory" version "4.9.8"
```

Comment:
To prevent duplication of things in sub-projects. There's quite an issue with how-to-properly define plugins in multi-module projects. But in general, if possible, they must be on top. And if necessary(like for Artifactory publishing target) - at the sub-project they're required in.
In short, `plugins {}` is preferred in sub-projects but if a plugin is exists in top project, version must be abandoned. `apply` is fine at top but can be dangerous some plugin applied more than once - `gradle` throws an exception in this case, telling that plugin is already in `classpath`.